packages/grafana-ui/src/components/Tooltip/Popper.tsx

```diff
@@ -13,45 +14,41 @@ const defaultTransitionStyles = {
   opacity: 0,
 };
 
-const transitionStyles = {
+const transitionStyles: {[key: string]: object} = {
   exited: { opacity: 0 },
   entering: { opacity: 0 },
   entered: { opacity: 1 },
   exiting: { opacity: 0 },
 };
 
-interface Props {
+interface Props extends React.DOMAttributes<HTMLDivElement> {
   renderContent: (content: any) => any;
   show: boolean;
-  placement?: any;
+  placement?: PopperJS.Placement;
   content: string | ((props: any) => JSX.Element);
   refClassName?: string;
```

Comment:
I think we can remove the refClassName-prop now?

Reply:
Tru dat! Fixed in 6e3225c29e399ce4d0837975773a2acb93cd61d7